### PR TITLE
Add Index Coop to /protocols

### PR DESCRIPTION
**Pull request by TheYoungCrews**

### CODE REVIEW

**Review by gentyana — APPROVED** [review]

File: protocols/index-coop/config.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "name": "Index Coop",
+    "icon": "logo.png",
+    "metadata": {
+        "pt": [
+            {
+                "chainId": 1,
+                "address": "0xf7906F274c174A52d444175729E3fa98f9bde285",
+                "integrationUrl": "https://indexcoop.com/products/high-yield-eth",
+                "description": "High Yield ETH (hyETH) is a diversified index targeting the highest ETH-denominated yields on Ethereum Mainnet"
+            }
```
Comment (manh-pendle) [inline protocols/index-coop/config.json:11]: Need `,`

File: protocols/index-coop/config.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            {
+                "chainId": 1,
+                "address": "0x6ee2b5E19ECBa773a352E5B21415Dc419A700d1d",
+                "integrationUrl": "https://indexcoop.com/products/high-yield-eth",
+                "description": "High Yield ETH (hyETH) is a diversified index targeting the highest ETH-denominated yields on Ethereum Mainnet"
+            }
```
Comment (manh-pendle) [inline protocols/index-coop/config.json:17]: Need `,`

**Review by manh-pendle — COMMENTED** [review]
Please reformat your `config.json` file correctly

**Comment by TheYoungCrews** [comment]
sorry @manh-pendle, added missing comma for each instance.

**Comment by leduythuccs** [comment]
Hi there, your json is still not a valid json file, i suggest you to double-check the file format via https://jsonformatter.org/

**Pull request merged**